jest.config.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  collectCoverageFrom: ['lib/**/*.js']
+  collectCoverageFrom: ['lib/**/*.js'],
+  "moduleNameMapper": {
+    "@eslint/eslintrc": "@eslint/eslintrc/dist/eslintrc-universal.cjs"
```

Comment:
Could you add a comment or link docs as to why this is needed for future reference?

Reply:
There is a issue with eslint 8 and some version of jest when you  have local/custom ESLint rules and is tested with Jest and ESLint Rule Tester, i read that in some cases you must to remapped it

i take the reference from this link https://giters.com/facebook/jest/issues/11100

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/100316982/164737849-526717a1-780d-4cf5-a046-e22d256918d7.png">

Reply:
👌🏻 Good find. I was referring to adding a code comment since it is an otherwise arbitrary change.

Something like:
```suggestion
    // Work-around for subpath imports.
    // See: https://github.com/facebook/jest/issues/11100#issuecomment-965832643
    "@eslint/eslintrc": "@eslint/eslintrc/dist/eslintrc-universal.cjs"
```